### DISCUSSION TITLE
what is your favorite movie?

### DISCUSSION THREAD

**Discussion by junko-ha** [NAIST-SE/AttractingContributors#835] [2023 Attraction Contributions]
Hello!
I like watching movies:)
If you recommend me your favorite movie, I'll watch it.
Please tell me your best one!
Anyone is ok.
https://github.com/junko-ha/Whats_your_fav_movie/blob/main/README.md
Please add to my README file.

**Comment by KKKKKXY** [comment]
I added my favorite movie into your list.(githubid: KKKKKXY)
